### PR TITLE
Paras's work for v1.2 (Up for Review Before Merge)

### PR DESCRIPTION
**Pull request by ParasK26**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewDueRemindersCommand.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.timeidentifiedclass.shopday.Reminder;
+
+/**
+ * This command allows us to view all reminders in the active shop day that are due.
+ */
+
+public class ViewDueRemindersCommand extends Command {
+
+    public static final String COMMAND_WORD = "duereminders";
```
Comment (amirulmaricar) [inline src/main/java/seedu/address/logic/commands/ViewDueRemindersCommand.java]: Command word could be shortened to one word for ease of use.
Possible words: due, reminders

Comment (amirulmaricar) [reply src/main/java/seedu/address/logic/commands/ViewDueRemindersCommand.java]: Generic terms might help users remember better

File: src/main/java/seedu/address/model/timeidentifiedclass/transaction/TransactionTest.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public static void main(String[] args) {
         } catch (ClosedTransactionException c) {
             System.out.println("Successfully caught ClosedTransactionException");
         }
-        System.out.println(transaction.getTransactionRecord());
+        System.out.println(transaction.getTransactionRecordAsString());
```
Comment (amirulmaricar) [inline src/main/java/seedu/address/model/timeidentifiedclass/transaction/TransactionTest.java]: System out could be implemented as an object/method itself

File: src/main/java/seedu/address/logic/commands/ViewDueRemindersCommand.java (hunk 1 above)
Comment (ParasK26) [reply src/main/java/seedu/address/logic/commands/ViewDueRemindersCommand.java]: Agreed. I need to think of a better word. Perhaps 'pending'?

**Pull request merged**